.vscode/launch.json

```diff
@@ -219,6 +219,24 @@
                 "set print pretty",
             ]
         },
+        {
+            "name": "Debug px4_fmu-v5X (jlink)",
```

Comment:
You can drop this part if you rebase on master. This entry can be shared with v5. The device (STM32F765VI) and svd are the same.
https://github.com/PX4/Firmware/commit/a462bfeb538f3f35af30c9296a52bf5e5dd223d3#diff-7fde71560fa9376d2d93a25b42d719f5

Reply:
Jup, you are right. I dropped the entry